src/core/src/pass/serialize.cpp

```diff
@@ -420,6 +421,38 @@ class XmlSerializer : public ov::AttributeVisitor {
         }
     }
 
+    void pack_string_tensor(ov::AttributeAdapter<std::shared_ptr<ov::StringAlignedBuffer>>* adapter,
+                            std::shared_ptr<uint8_t>& data,
+                            size_t& total_size) {
+        // packed format is the following:
+        // <num_string>, <1st string offset>,..., <nth string offset>, <1st string raw format>,..., <nth rawformat>
+        auto num_elements = adapter->get()->get_num_elements();
+        auto strings = reinterpret_cast<std::string*>(adapter->get()->get_ptr());
+
+        // first run over all elements: calculate total memory required to hold all strings
+        size_t symbols_size =
+            std::accumulate(strings, strings + num_elements, size_t(0), [](size_t accum, const std::string str_ptr) {
+                return accum + str_ptr.size();
+            });
+        total_size = 4 * (1 + 1 + num_elements) + symbols_size;
```

Comment:
Could you add some `constexpr` variable which explain adding `1+1` to the `num elements`?